src/foreman.ts

```diff
@@ -47,7 +48,13 @@ function chooseAsset(release: GitHubRelease): GitHubAsset | null {
       name.includes("win64") ||
       name.includes("win32");
   } else if (process.platform === "darwin") {
-    platformMatcher = name => name.includes("macos");
+    const arch = os.arch();
+    if (arch === "x64") {
+      platformMatcher = name => name.includes("macos-x86_64");
+    } else {
+      platformMatcher = name => name.includes("macos-arm64");
+    }
```

Comment:
We may still need some sort of fallback here since there are older projects with just a `macos` asset. I presume that would likely be only x86_64 ones anyway, so maybe the x64 case needs to include a `|| name.includes("macos")` or something?

Reply:
Oh wait, this is only for foreman itself, huh. I wonder if we can retroactively fix up the asset names?